client/js/components/procedure/StatementSegmentsList/SegmentsRecommendations.vue

```diff
@@ -188,9 +196,33 @@ export default {
         })
     },
 
-    fetchSegments () {
+    async fetchSegments () {
```

Comment:
nitpick: this function should be renamed because it does a lot more than fetching segments